src/commands/test/gen_thread_test.ts

```diff
@@ -39,7 +39,7 @@ if (process.send) {
                 //#region Test
                 const sTime = Date.now();
                 for (let i = 0; i < args.it; i++) {
-                    const key = gen[args.prefix][args.postfix](session, args.token);
+                    const key = gen[args.prefix][args.postfix](session, args.token, true);
```

Comment:
Not a huge fan of this. We may have to change some of their OG code in `gen_helper.ts` to accommodate.
We should pass the name in at this point, rather than the `test` bool that really just changes the name.
More to come in the `gen_helper` file.